debian/mgmt.service

```diff
@@ -0,0 +1 @@
+../misc/mgmt.service
```

Comment:
@dsx no nood to call manually dsx_. It it is in install that well be called automatically.